src/components/Typography/Typography.less

```diff
@@ -0,0 +1,54 @@
+@import (reference) '../../styles/variables.less';
+@import (reference) '../../styles/mixins.less';
+
+.@{prefix}-Typography {
+
+	&-h1,
+	&-h2,
+	&-h3 {
+		margin: 0;
+		font-weight: @font-weight-medium;
+		text-rendering: optimizelegibility;
+	}
+
+	// These values have been adjusted for the Xandr branding. Anything after h3
+	// shouldn't be used in practice.
+	&-h1 {
+		font-size: 24px;
+		line-height: 28px;
+	}
+	&-h2 {
+		font-size: 16px;
+		line-height: 20px;
+	}
+	&-h3 {
+		font-size: 12px;
+		line-height: 16px;
+		text-transform: uppercase;
+		color: @color-neutral-6;
+	}
+
```

Comment:
there doesn't seem to be a style for `&-p`, should there be?

Reply:
I added an explicit statement in the less file for it. Not much to it, but it's there.